src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -78,12 +78,11 @@
         >
           <span>{{ channelName }}</span>
         </router-link>
-        <span
-          v-if="!isLive && !hideViews"
-          class="viewCount"
-        >• {{ parsedViewCount }}</span>
-        <span v-if="viewCount === 1">{{ $t("Video.View") }}</span>
-        <span v-else-if="parsedViewCount !== ''">{{ $t("Video.Views").toLowerCase() }}</span>
+        <template v-if="!isLive && !hideViews">
```

Comment:
Not sure why you're using a template here.  Please switch this to something else like a `div` or another `span` unless I'm missing something here.

Reply:
I'm only doing so the final result looks like this:
```html
<span class="viewCount">• (parsedViewCount here)</span>
<span>(View / Views string here)</span>
```
instead of this (example with div):
```html
<div>
  <span class="viewCount">• (parsedViewCount here)</span>
  <span>(View / Views string here)</span>
<div>
```
since the template tag gets interpreted as an invisible wrapper here
https://vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt

I assume you already know this, but I'm not 100% sure, so let me know if you still want a div or span to wrap them instead @PrestonN

Reply:
Ahh, that's fair.  It just wasn't something I used too often, so I'm slightly unfamiliar with that method.  Yeah this is perfectly fine, no worries.